.vscode/launch.json

```diff
@@ -12,7 +12,7 @@
       "program": "${workspaceFolder}/build/test/manifold_test",
       "args": [
         "--gtest_catch_exceptions=0",
-        "--gtest_filter=Manifold.Simplify"
+        "--gtest_filter=SDF.SineSurface"
```

Comment:
This test is passing, but I think that's just because it's not tight enough; it looks wrong when I export it. I need to check on that before merging this.

Reply:
I'm thinking does this mean that we should have some image based check in our CI (also available locally), similar to what openscad does (https://github.com/openscad/openscad/blob/master/tests/image_compare.py).